File: onboarding-servey/src/main/java/com/innercircle/onboardingservey/infrastructure/repository/AnswerRepository.java
package com.innercircle.onboardingservey.infrastructure.repository;

import com.innercircle.onboardingservey.domain.model.entity.Answer;
import com.innercircle.onboardingservey.domain.model.entity.Survey;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AnswerRepository extends JpaRepository<Answer, Long> {
    List<Answer> findAnswersBySurvey(Survey survey);
}

File: src/main/kotlin/org/inner/circle/survey/infrastructure/repository/AnswerRepository.kt
package org.inner.circle.survey.infrastructure.repository

import org.inner.circle.survey.domain.entity.Answer
import org.inner.circle.survey.domain.entity.Survey
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface AnswerRepository : JpaRepository<Answer, Long> {
    fun findAllBySurvey(survey: Survey): List<Answer>?
}
